pic_cxxflags.patch

From: Christian Prochaska <christian.prochaska@genode-labs.com>

Allow passing of PIC_CXXFLAGS to the configure script of libstdc++.
Without this change, libsupc++ would be compiled w/o PICFLAGS, resulting
in text relocations. Because for base tool chains, no 'dynamic_linker'
is defined (see 'libtool.m4'), 'dynamic_linker' is set to 'no', which
results in 'can_build_shared = no', which, in turn , results
in 'enable_shared = no', which, in turn, sets 'PIC_CXXFLAGS' to
nothing rather then '-prefer-pic'.
---
 libstdc++-v3/configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libstdc++-v3/configure.ac b/libstdc++-v3/configure.ac
index f04563e..238097d 100644
--- a/libstdc++-v3/configure.ac
+++ b/libstdc++-v3/configure.ac
@@ -424,7 +424,7 @@ GLIBCXX_EXPORT_FLAGS
 if test "$enable_shared" = yes; then
   PIC_CXXFLAGS="-prefer-pic"
 else
-  PIC_CXXFLAGS=
+  __PIC_CXXFLAGS=
 fi
 AC_SUBST(PIC_CXXFLAGS)
 
